zeroconf.py

```diff
@@ -39,7 +39,7 @@
 
 __author__ = 'Paul Scott-Murphy, William McBrine'
 __maintainer__ = 'Jakub Stasiak <jakub@stasiak.at>'
-__version__ = '0.21.0'
+__version__ = '0.21.1'
```

Comment:
This is out of scope, let us handle that externally.

Reply:
Sure, I'll take that out.